jared-alrightjones/Homework-4/madlib-with-loops/js/app.js

```diff
@@ -0,0 +1,24 @@
+// provided code
+
+var startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+var startupY = ['Slack', 'GitHub', 'Tesla', 'Hyperloop', 'Harvest'];
+
+var random1 = Math.floor((Math.random() * startupX.length));
+var random2 = Math.floor((Math.random() * startupY.length));
+
+// create something
+
+var greatIdea;
+const createBtn = document.querySelector('#create'); 
+const outputField = document.querySelector('#xForY');
+createBtn.addEventListener('click' outputText)
+
+function outputText() {
+	greatIdea = 'A startup that is ' + startupX[random1] + ', but for ' + startupY[random2];
+	startup.innerHTML = startupIdea;
```

Comment:
`startup` is not the name of a variable you created. Which element you selected above should get this text as its content?